### ISSUE TITLE
Crash on Startup - Unknown windowLayoutInDisplayCutoutMode: 3

### ISSUE THREAD

**Issue by cw2k** [Leonidius20/RecordingStudio#11]
On Samsung S9 I get welcomed with the crashscreen:
**Unknown windowLayoutInDisplayCutoutMode: 3**

Version 0.1.2 works except for it's known bug to not showing its recordings.
```
Build version: 0.1.3 
Current date: 2024-09-30 08:33:18 
Device: Samsung SM-G960F 
OS version: Android 10 (SDK 29) 
 
Stack trace:  
java.lang.RuntimeException: Unable to start activity ComponentInfo{io.github.leonidius20.recorder.lite/io.github.leonidius20.recorder.MainActivity}: 
java.lang.UnsupportedOperationException: Unknown windowLayoutInDisplayCutoutMode: 3
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3654)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3806)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2267)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:8167)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)
Caused by: java.lang.UnsupportedOperationException: Unknown windowLayoutInDisplayCutoutMode: 3
	at com.android.internal.policy.PhoneWindow.generateLayout(PhoneWindow.java:2693)
	at com.android.internal.policy.PhoneWindow.installDecor(PhoneWindow.java:2894)
	at com.android.internal.policy.PhoneWindow.getDecorView(PhoneWindow.java:2197)
	at io.github.leonidius20.recorder.MainActivity.onCreate(SourceFile:28)
	at android.app.Activity.performCreate(Activity.java:7963)
	at android.app.Activity.performCreate(Activity.java:7952)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1307)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3629)
	... 11 more
`
```

I really appreciate your efforts developing a handy audio recording app on Android.

One year I used an IPhone and love the recording app that comes with iOS.
Now tested many recording apps on Android and they are all in some way laking features or are simply crap.

Important features:
1. Easy start, stop or cancel recording. I don't like to be ask after stop the recording to enter some filename and have to tap 3 or 4 times on the screen to start a new recording. 
Yes I have seen crap like this man. Asking ppl for some audio quality/profil before recording. Also had some conversation with a dev to disable that question to disable that 'Do you like to cancel the recording?'. ... or add an option to do so.

No it's a security feature. Okay carry on dreaming and sorry for wasting our time, I move on to some other app. Do you use the apps yourself you develop? Probably not.

Okay be able to enter/edit the filename while recording is great, but not may app devs though in the practical way.

...and yes adding bookmarks with notes is a nice useful addon.

2. Support for ID3 tags at least for some useful fields.

3. Keep recording in chronical order and the ability to and tags to them / filter by tags.

4. Easy merge two or more audio files as well as delete and rename them

5. Basic audio editor that is also available to other apps via share with feature. Save changes should be able to overwrite the original file.  Actual anti example Lexis Audio Editor. it opens sound files via sharing but ask for filename on save. Navigating to the source file it says can't overwrite file while giving giving no option to delete a file. Now I can use a third app like xplor to do that file deleting job. However how complicated and time wasteful is that.

7. Audio Editor features: 
cut and trim audio files !!!
Man there are so many sound recorder there which just can't do. Lacking of doing so you degrades them to 'toy' apps. Example:
Neutron audio recorder.
Great app to demonstrate their powerful audio driver. The EQ and AGC automatic gain control or more understandable volumen regulation is outstanding. However the UI is loke crap and tooked me a day to learn to fully understand and if the recording is done you sit there with the file. No way to edit or alter it. Shall I play and re-record it? Good ideas not brought to an end / useful solution.

Okay an absolute supreme plus for the audio editor that would make your audio recorder zo a kill app would be a
Normaliser
And maybe a good useful pre-set for DRC. Dynamic Range compressor in form of an downward compressor/hardlimitor that pushes up volume while trying to avoid clipping.

**Comment by Leonidius20** [comment]
Hello and thank you for reporting this issue.
It will be fixed in the next version, I'll comment here when it becomes available. The recordings list is already fixed in 0.1.3, if you could get to the recordings list that is😎

As for the normalizer/compressor - the killer feature and the motivation for creating this app was to add support for external audio effect plugins. This will be my top priority after releasing version 0.2.0, which is currently in the making. Then these effects could be implemented as third party plugins. I was planning to implement a few plugins myself, maybe it could be the ones you listed. But it will take a while to make this functionality happen.

I am planning to add cut and trim, but only after the effects are implemented (or while they are being implemented maybe)

I will think about the ID3 and the merge functionality

**Comment by Leonidius20** [comment]
Version 0.1.4 has just been released and it should fix the crash. It is now available in the Releases section. Please check, thank you
